support/fakeclientwithscheme.go

```diff
@@ -0,0 +1,11 @@
+package support
+
+import (
+	"k8s.io/apimachinery/pkg/runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+)
+
+func NewFakeKubeClientWithScheme(scheme *runtime.Scheme, objects ...client.Object) client.Client {
```

Comment:
Can you move it to `fakeclient.go`. There is no need to have separate files for similar functions.

Reply:
Done